### DISCUSSION TITLE
Non-Docker Migration - GeoIP / AS Numbers in Clickhouse

### DISCUSSION THREAD

**Discussion by washcroft** [akvorado/akvorado#1466] [Q&A] [answered]
Hello,
Recently updated Akvorado to the latest version, but since doing so data isn't being enriched with GeoIP/AS data.

I noticed this in the release notes of v1.10.0:

> On this release, geo IP is now performed in ClickHouse instead of inlet. When
> using the standard docker compose setup, the configuration should be
> automatically migrated from the inlet component to the clickhouse component.
> This also changes how geo IP is used for AS numbers: geo IP is used as last
> resort when configured. It also increases memory usage (1.3GB for ClickHouse).

What needs to be done to migrate non docker setups?

Thanks!

**Comment by vincentbernat** [comment, marked as answer]
The migration is about moving GeoIP from the inlet to the orchestrator. If your configuration files are served by the orchestrator, the configuration is also migrated (you can check by dumping the configuration of the orchestrator). If not, you need to move the configuration manually. The other piece is that you need the orchestrator to have the GeoIP files for ClickHouse to get them.

If this does not work, check:
- you have a `network` dictionary in ClickHouse (`show create table networks`)
- it is not empty (`select count(*) from networks`), and if it is, check ClickHouse errors (`select * from system.errors \G`)

**Reply by washcroft** [reply]
Thank you, resolved this by:

- Moving the geoip related config from the inlet config file to the to orchestrator config file, noting the file path keys have changed to lists.
- Ensuring the geoip files exist and are accessible to the orchestrator.
